packages/application-generic/src/factories/push/handlers/one-signal.handler.ts

```diff
@@ -8,13 +8,18 @@ export class OneSignalHandler extends BasePushHandler {
   }
 
   buildProvider(credentials: ICredentials) {
-    if (!credentials.apiKey || !credentials.applicationId) {
+    if (
+      !credentials.userKey ||
+      !credentials.applicationId ||
+      !credentials.appKey
+    ) {
       throw Error('Config is not valid for OneSignal');
     }
 
     this.provider = new OneSignalPushProvider({
       appId: credentials.applicationId,
-      apiKey: credentials.apiKey,
+      appKey: credentials.appKey,
+      userKey: credentials.userKey,
```

Comment:
Unfortunately this change is causing a breakin change for all of one signal users, is there some mapping between the old apiKey to the new appKey? is that the same value?